starter-code/scripts/article.js

```diff
@@ -13,18 +20,26 @@ Article.prototype.toHtml = function() {
   However, in our modules.css stylesheet, we gave all elements
   with a class of template a display of none. Let's make
   sure we're not accidentally hiding our cloned article! */
+  // Done
+  $newArticle.removeClass('template');
 
   if (!this.publishedOn) $newArticle.addClass('draft');
   $newArticle.data('category', this.category);
 
   /* TODO: Now use jQuery traversal and setter methods to fill in the rest
   of the current template clone with properties from this particular Article instance.
   We need to fill in:
-    1. author name,
+
     2. author url,
     3. article title,
     4. article body, and
     5. publication date. */
+  $newArticle.find('.Byline a').html(this.author);
+  $newArticle.find('.Byline a').attr('href', this.authorUrl);
+  $newArticle.find('#articles h1').html(this.title);
```

Comment:
This is not populating on your page as well. Do you need the id? Because the template is off of the class=template, which is below the id=articles in the hierarchy of the html.